### ISSUE TITLE
"updaterate" not working properly

### ISSUE THREAD

**Issue by staviq** [ldtteam/minecolonies#4945]
TL;DR increasing "updaterate" makes villagers stop working.

minecolonies-0.11.918-BETA-universal
forge 1.15.2 - 31.1.77

I tried to bump up the speed of things a bit since i'm playing singleplayer and have some cpu cycles to spare.

The description of "updaterate" is not very informative, so i just tried to set it to max - 100, and after i quit the game yesterday and started it back today, every villager was stuck not doing anything.

I did cancel a build order, but the builder thingy in builders hut was still showing missing resources for the canceled build. The townhall thingy was showing no work orders at all.

I changed the "updaterate" back to default and after a restart everything is back to normal.

Absolutely no errors or any log entries related to minecolonies in the log file.

**Comment by someaddons** [comment]
increasing updaterate increases cpu/server performance, not the villagers ones. It makes villagers skip updates

**Comment by staviq** [comment]
Can you please just modify config description with that exact explaination ?

**Comment by someaddons** [comment]
ye
